W-14955232-MMPmuleVersion-dm (#2623)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/deploy-to-cloudhub.adoc

| `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
Example value: `4.3.0` +

> Starting with Mule 4.5, deployments to CloudHub accepts the Mule version exact name or the semantic versioning, which deploy the latest version of Mule runtime. +

Reviewer: ```suggestion
Starting with Mule 4.5, deployments to CloudHub accept the Mule version exact name or the semantic versioning, which deploys the latest version of Mule runtime. +
```

Reviewer: I assume the plus at the end of the line in a table is a line break?

Author: It is!

Example values: `4.6`, `4.6-java8` , `4.6-e-java17` +

> When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection may cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`lts` or `e`) and `Java` (`java8` or `java17`) properties to define deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven plugin 4.1.0 Release Notes]. | Yes

Reviewer: ```suggestion
When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`lts` or `e`) and `Java` (`java8` or `java17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes
```

Reviewer: define -> specify?

> When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`lts` or `e`) and `Java` (`java8` or `java17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes

Author: ```suggestion
When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`NONE`, `EDGE`, or `LTS`) and `javaVersion` (`8` and `17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes
```

> When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`NONE`, `EDGE`, or `LTS`) and `javaVersion` (`8` or `17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes

Author: ```suggestion
When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes
```

> When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`NONE`, `EDGE`, or `LTS`) and `javaVersion` (`8` and `17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes

Author: ```suggestion
When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, using the `muleVersion` property to specify the release channel and Java version selection might cause unexpected deployments. For example, specifying `<muleVersion>4.6-lts-java17</muleVersion>` in the plugin 3.8.0 or 4.0.0 deploys the application with an Edge release channel and a Java 8 selection. To specify the release channel and Java version, upgrade the plugin to version 4.1.0 and use the `releaseChannel` (`NONE`, `EDGE`, or `LTS`) and `javaVersion` (`8` or `17`) properties to define the deployment selection. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes]. | Yes
```

> | `releaseChannel`| Set the name of the release channel to be used for the selected Mule version. Supported values are `NONE`, `EDGE`, and `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, you get an error. | No

Reviewer: ```suggestion
| `releaseChannel`| Set the name of the release channel to use for the selected Mule version. Supported values are `NONE`, `EDGE`, and `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, an error occurs. | No
```

> | `javaVersion` | Set the name of the Java version to be used for the selected Mule version. Supported values are `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected Java version is used. If the Java version you select is not available for the specified Mule version, you get an error. | No

Reviewer: ```suggestion
| `javaVersion` | Set the name of the Java version to use for the selected Mule version. Supported values are `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected Java version is used. If the Java version you select isn't available for the specified Mule version, an error occurs. | No
```

| `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
Example value: `4.3.0` +
Starting with Mule 4.5, deployments to CloudHub accepts the Mule version exact name or the semantic versioning, which deploy the latest version of Mule runtime. +

> Example values: `4.6`, `4.6-java8` , `4.6e-java17` | Yes

Reviewer 2: We also need 2 new rows for 1) javaVersion 2) releaseChannel fields -- which is missing at this point
